src/Symfony/Bridge/Doctrine/ManagerRegistry.php

```diff
@@ -46,7 +46,7 @@ protected function resetService($name)
         $manager = $this->container->get($name);
 
         if (!$manager instanceof LazyLoadingInterface) {
-            throw new \LogicException(sprintf('Resetting a non-lazy manager service is not supported. Set the "%s" service as lazy and require "symfony/proxy-manager-bridge" in your composer.json file instead.', $name));
+            throw new \LogicException(sprintf('Resetting a non-lazy manager service is not supported. Require "symfony/proxy-manager-bridge" in your composer.json file to set the "%s" service as lazy.', $name));
```

Comment:
If the service is already declared as lazy, then requiring the bridge is the only missing step to make it truly lazy. That's what happened to you here with the EM. I'd suggest this instead:
`'Resetting a non-lazy manager service is not supported. '.(interface_exists(LazyLoadingInterface::class) ? sprintf('Declare the "%s" service as lazy.', $name) : 'Try running "composer require symfony/proxy-manager-bridge".'`